packages/blockchain/src/db/operation.ts

```diff
@@ -21,10 +21,10 @@ export enum DBTarget {
   HashToNumber,
   NumberToHash,
   TotalDifficulty,
-  CliqueSignerStates,
-  CliqueVotes,
   Body,
   Header,
+  CliqueSignerStates,
+  CliqueVotes,
```

Comment:
Totally ok with this change but out of curiosity: is the change of ordering here a (potential) problem?

Reply:
I was debugging some db issues and thought it might have been, but in the end I don't think it was a problem.

I kept it this way though because otherwise it changed the enum integers for `Body, Header` and figured it would be better to maintain the existing numbering (and also found it a bit easier to read having them separate at the bottom)